crates/emmylua_code_analysis/resources/std/global.lua

```diff
@@ -243,7 +250,7 @@ function pairs(t) end
 ---@generic T, R, R1
 ---@param f fun(...: T...): R1, R...
 ---@param ... T...
----@return boolean, R1|string, R...
+---@return boolean, R1|any, R...
```

Comment:
Here and in `xpcall`, it would be better to have a separate type like `unknown`, because `any` is too broad.